### PR TITLE
Add workflow to deploy to production

### PR DESCRIPTION
**Pull request by howard-e**
Add workflow to automate deployment to aria-at.w3.org on a push to the `main` branch.

### CODE REVIEW

File: .github/workflows/deploy-production.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+jobs:
+  deploy-production:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install SSH key for deploying
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.DEPLOY_SSH_PRIVATE_KEY }}
+          known_hosts: ${{ secrets.DEPLOY_KNOWN_HOSTS_PRODUCTION }}
```
Comment (alflennik) [inline .github/workflows/deploy-production.yml:17]: This variable doesn't seem to be defined in GitHub! Is that expected?

**Review by alflennik — COMMENTED** [review]
Looks great, this type of automation will make the project a lot more approachable and easier to manage. See my one comment but other than that looks good.

File: .github/workflows/deploy-production.yml (hunk 1 above)
Comment (howard-e) [reply .github/workflows/deploy-production.yml:17]: Added!

**Review by alflennik — APPROVED** [review]
Thanks for the fix!

**Pull request merged**